src/backend/crew_ai/crew.py

```
@@ -2,13 +2,59 @@
 import re
 import logging
 
 logger = logging.getLogger(__name__)
 
+# CrewAI log file path and rotation settings
+CREWAI_LOG_FILE = "logs/crewai.log.txt"  # CrewAI appends .txt to paths not ending in .json/.txt
+CREWAI_LOG_MAX_BYTES = 50 * 1024 * 1024  # 50MB per file
+CREWAI_LOG_BACKUP_COUNT = 9              # 9 backups = 450MB max
+
+
+def _rotate_crewai_log():
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_rotate_crewai_log`

Add return type annotation: `None`

(ANN202)